app/Http/Requests/UpdateArtist.php

```diff
@@ -24,7 +24,6 @@ public function authorize()
     public function rules()
     {
         return [
-            'moniker' => 'required|max:255', // TODO: removing `required` from this causes tests to fail ?
```

Comment:
Shouldn't this remain here ? Just ***not*** `required` ?

Reply:
Oh I read the commit body ... sorry for missing that

Reply:
That's a good point; even though Catalog Entities (Artists and Labels) aren't permitted to change their own monikers (they must submit a request for same, which a staff member must review prior to accepting the change), the validation rules should still apply. I'll fix this now.